services/appflowy-worker/src/import_worker/worker.rs

```diff
@@ -1310,22 +1322,41 @@ async fn upload_file_to_s3(
 }
 
 async fn get_encode_collab_from_bytes(
+  workspace_id: &str,
   object_id: &str,
   collab_type: &CollabType,
   pg_pool: &PgPool,
+  s3: &Arc<dyn S3Client>,
 ) -> Result<EncodedCollab, ImportError> {
-  let bytes = select_blob_from_af_collab(pg_pool, collab_type, object_id)
-    .await
-    .map_err(|err| ImportError::Internal(err.into()))?;
-  tokio::task::spawn_blocking(move || match EncodedCollab::decode_from_bytes(&bytes) {
-    Ok(encoded_collab) => Ok(encoded_collab),
-    Err(err) => Err(ImportError::Internal(anyhow!(
-      "Failed to decode collab from bytes: {:?}",
-      err
-    ))),
-  })
-  .await
-  .map_err(|err| ImportError::Internal(err.into()))?
+  let key = collab_key(workspace_id, object_id);
+  match s3.get_blob_stream(&key).await {
+    Ok(mut resp) => {
+      let mut buf = Vec::with_capacity(resp.content_length.unwrap_or(1024) as usize);
+      resp
+        .stream
+        .read_to_end(&mut buf)
+        .await
+        .map_err(|err| ImportError::Internal(err.into()))?;
+      let decompressed = zstd::decode_all(&*buf).map_err(|e| ImportError::Internal(e.into()))?;
```

Comment:
will `get_encode_collab_from_bytes` causes tokio runtime issue If it takes 10 ms to decode the buf?